lib/matplotlib/backends/backend_ps.py

```diff
@@ -361,9 +361,9 @@ def create_hatch(self, hatch):
 
      /PaintProc {{
         pop
-        {linewidth:g} setlinewidth
-{self._convert_path(
-    Path.hatch(hatch), Affine2D().scale(sidelen), simplify=False)}
+        {linewidth:f} setlinewidth
+        {self._convert_path(
+            Path.hatch(hatch), Affine2D().scale(sidelen), simplify=False)}
```

Comment:
Can you back these changes all the way out?

Reply:
These changes were done as flake8 failed while committing

Reply:
flake8 should not have flagged either the g->f change or indentation in a multi-line string.  

If flake8 is indeed triggering and requiring either of these changes that is a bug is flake8.

Reply:
I've committing after skipping flake8, can't do anything else I think